internal/gengapic/metadata.go

```diff
@@ -30,6 +30,7 @@ func (g *generator) genGapicMetadataFile() error {
 }
 
 func (g *generator) addMetadataServiceForTransport(service, transport, lib string) {
+	// This method is idempotent.
```

Comment:
Generally comments for functions should go right above the function.

https://golang.org/doc/effective_go#commentary